src/lib/NoncedStyleElement.jsx

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+/** @jsx node */
+
+import { node, type ChildType, type NullableChildrenType } from 'jsx-pragmatic/src/node';
+import { Style } from 'jsx-pragmatic/src/component';
+
+type StyleProps = {|
+    css : string | {| _getCss : () => string |},
+    nonce : ?string,
```

Comment:
Is this what we really want here?  Is it ok if the `nonce` value is null or undefined?  Or do we want to enforce a value other than null or undefined with `nonce : string,`?